### DISCUSSION TITLE
Welcome to Groupe-1 Discussions!

### DISCUSSION THREAD

**Discussion by Mohamed-ux-lab** [Mohamed-ux-lab/Groupe-1#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mohamed-ux-lab** [comment]
Salut tout le monde j'espere que tout vas bien avec vos git et github en cas de probleme vous me faite signe ok !!!.

**Comment by Mohamed-ux-lab** [comment]
Pour le travail on vas le repartir de la sorte :
-D'abord tout le monde modifie un article sur la page;
-Ensuit Moh tu t'occupe du sidebar;
-Awa tu modifie la couleur de la page ;
-Dicko toi tu vas modifier la police de la page entière ;
-Sanago vas modifier le header en centrant la nav et en effaçant les icone facebook et twitter ;
-Yaya toi tu vas bien Modifier le footer en ajoutant 'créer par le groupe-1' et tu y mettra les icones facebook et twitter.
Voila c'est tout pour la répartition du travail donc bonne chance à tous !!!!.

**Reply by Scorpionkarabenta** [reply]
D’abord tout le monde modifie un article sur la page.
je n'arrive pas encore à moitié mon article.

**Reply by Scorpionkarabenta** [reply]
aide-moi koureissi

**Reply by Mohamed-ux-lab** [reply]
C'est dans le fichier 📂 index.html et tu cherche les balise <article> et tu modifie le contenu.

**Reply by Scorpionkarabenta** [reply]
ok
